framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java

```diff
@@ -41,7 +41,7 @@ public ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl() {
 
         ProjectSpecificPropertyNames propertyNames = ProjectSpecificPropertyNames.create();
         envVariables.put(propertyNames.workbenchHttpsSecret(), OpenShiftConstants.getKieApplicationSecretName());
-        envVariables.put(propertyNames.workbenchMemoryLimit(), "4Gi"); // limit memory limit to use only 4Gi
+        envVariables.put(propertyNames.workbenchMemoryLimit(), "2Gi"); // limit memory limit to use only 4Gi
```

Comment:
According to my testing, it seems to be working fine with 2Gi and with this constraint I guess tests will require less to work. However, let me know if you want it to keep it with 4Gi.

Reply:
If it works with 2Gi then it is ok.